src/yarpmotorgui/main.cpp

```diff
@@ -138,12 +139,16 @@ int main(int argc, char *argv[])
         desc_driver->view(idesc);
         if (idesc)
         {
-            idesc->getControlBoardWrapperDevices(robot_description);
-            for (size_t i = 0; i < robot_description.size(); i++)
+            idesc->getAllDevicesByType("controlboardwrapper", cbw1_list);
```

Comment:
I guess that with `controlboardwrapper` you mean the former version of `controlboardwrapper2`. 
In this case the right name is `controlboard`, but it is _formally deprecated_ since the last YARP release, so I think we can safely ignore them.